dev-docs/a2-chef-server.md

```diff
@@ -0,0 +1,152 @@
+concrete plans or schedule. Deploying HA chef server via the Automate deployment service
+is intended as a services option only.
+
+As of July 2019, the audience for the chef server deployed via the Automate deployment service consists
+of:
+
+1. opsworks, which deploys an all-in-one Automate with chef server on a single node
+2. the Customer Success team, which in July 2019 will start using the deployment service to deploy HA
+   chef server at selected customer sites,
+3. the Professional Services team, which is planning to start using the deployment service
+   to deploy HA chef server some time around August 2019.
+
+This document was written with the latter two groups in mind.
+
+## Deploy a new standalone chef server via Automate
+1. Install the chef-automate CLI tool. You will need one that contains commits: look for
+   hashes.
+2. Run `chef-automate init-config` to generate an editable config.toml.
+3. Add the following to the config.toml from 2:
+```
+[erchef.v1.sys.data_collector]
+enabled = false
+```
+Make any other config changes necessary.
+4. Run `chef-automate deploy --product chef-server config.toml` to install the chef
+   server. When the chef server comes up, the running services are:
+```
+   $ sudo ./chef-automate status
+
+   Status from deployment with channel [dev] and type [local]
+
+   Service Name            Process State  Health Check  Uptime (s) PID
+   deployment-service      running        ok            400        12974
+   backup-gateway          running        ok            280        13361
+   automate-postgresql     running        ok            277        13464
+   automate-pg-gateway     running        ok            280        13354
+   automate-elasticsearch  running        ok            279        13399
+   automate-es-gateway     running        ok            274        13706
+   pg-sidecar-service      running        ok            274        13715
+   es-sidecar-service      running        ok            258        14344
+   license-control-service running        ok            275        13650
+   automate-cs-bookshelf   running        ok            273        13771
+   automate-cs-oc-bifrost  running        ok            271        13857
+   automate-cs-oc-erchef   running        ok            258        14352
+   automate-cs-nginx       running        ok            270        13920
+   automate-load-balancer  running        ok            267        14166
+```
+
+## Forward data from the Automate-deployed chef server to Automate
+1. Make sure you have the data collector token from your Automate instance handy.
+2. On the node where you have deployed the chef server, create a chef_server_patch.toml file that contains
+```
+   [global.v1.external.automate]
+   enable = true
+   endpoint = "https://automate.example.com/data-collector/v0/" # is this right?
+   [global.v1.external.automate.auth]
+   token = "yourdatacollectortoken"
+
+   [deployment.v1.svc]           # is this section necessary?
```

Comment:
not for a patch since it'll already be configured, but like, it ain't gonna hurt anything by being there.